Framework/API/src/LogManager.cpp

```diff
@@ -146,17 +147,19 @@ const Types::Core::DateAndTime LogManager::startTime() const {
     } catch (std::invalid_argument &) { /*Swallow and move on*/
     }
   }
+  try {
```

Comment:
Rather than try/catch (which is generically more expensive), using `hasProperty("proton_charge")` would achieve the same result.

Reply:
Done.